---
layout: post
title:  "VS ribozyme"
date:   2022-04-18T14:25:52-05:00
author: Chichau Miao
categories: ribozyme
---


**Rfam ID:na** <br>

#### click into different sections:

- [Timeline](#timeline)
- [Description](#description)
- [Structure](#structure)
- [Reference](#reference)


## Timeline

***
<html lang="zh-cn">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>

</head>
<style>
  table {
        border: 2px solid #f8f8ff;
        border: 2px solid #767676;
		    border: 2px solid #767676;
		    border-radius: 5px;
		    background-color: #fff;
        }
.timeline {
  list-style: none;
  padding: 10px 0 10px;
  position: relative;
}
.timeline:before {
  top: 0;
  bottom: 0;
  position: absolute;
  content: " ";
  width: 3px;
  background-color: #eeeeee;
  left: 50%;
  margin-left: -1.5px;
}
.timeline > li {
  margin-bottom: 10px;
  position: relative;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li > .timeline-panel {
  width: 46%;
  float: left;
  border: 1px solid #d4d4d4;
  border-radius: 2px;
  padding: 5px;
  position: relative;
  -webkit-box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
}
.timeline > li > .timeline-panel:before {
  position: absolute;
  top: 16px;
  right: -12px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #ccc;
  border-right: 0 solid #ccc;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-panel:after {
  position: absolute;
  top: 16px;
  right: -10px;
  display: inline-block;
  border-top: 9px solid transparent;
  border-left: 10px solid #fff;
  border-right: 0 solid #fff;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-badge {
  color: #fff;
  width: 25px;
  height: 25px;
  line-height: 40px;
  font-size: 1.4em;
  text-align: center;
  position: absolute;
  top: 16px;
  left: 48.5%;
  margin-left: 0px;
  background-color: #999999;
  z-index: 100;
  border-top-right-radius: 50%;
  border-top-left-radius: 50%;
  border-bottom-right-radius: 50%;
  border-bottom-left-radius: 50%;
}
.timeline > li.timeline-inverted > .timeline-panel {
  float: right;
}
.timeline > li.timeline-inverted > .timeline-panel:before {
  border-left-width: 0;
  border-right-width: 15px;
  left: -15px;
  right: auto;
}
.timeline > li.timeline-inverted > .timeline-panel:after {
  border-left-width: 0;
  border-right-width: 14px;
  left: -14px;
  right: auto;
}
.timeline-badge.primary {
  background-color: #2e6da4 !important;
}
.timeline-badge.success {
  background-color: #3f903f !important;
}
.timeline-badge.warning {
  background-color: #f0ad4e !important;
}
.timeline-badge.danger {
  background-color: #d9534f !important;
}
.timeline-badge.info {
  background-color: #5bc0de !important;
}
.timeline-title {
  margin-top: 0;
  color: inherit;
}
.timeline-body > p,
.timeline-body > ul {
  margin-bottom: 0;
  padding-bottom: 0;
}
.timeline-body > p + p {
  margin-top: 0px;
}

</style>
    <ul class="timeline">
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/2160856" target="_blank" style="font-size:25px;" >1990</a>  Discovery[1]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/7753865" target="_blank" style="font-size:25px;" >1995</a>  Secondary structure[2]</h4>
            </div>
            <div class="timeline-body">
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/11250907"  target="_blank" style="font-size:25px;">2001</a>  Importance of the 2-3-6 helical junction[3]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/11575922" target="_blank" style="font-size:25px;">2001</a>  A730 loop is important[4]</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/12368096" target="_blank" style="font-size:25px;">2002</a>  A756 is critical for catalysis[7]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/17464286" target="_blank" style="font-size:25px;">2007</a>  G638 is critical for catalysis[10]</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/18786398" target="_blank" style="font-size:25px;">2008</a>  SAXS-deriverd structure[11]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/18786398" target="_blank" style="font-size:25px;">2009null</a> The soixante-neuf experiment[12]</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/20547881" target="_blank" style="font-size:25px;">2010</a>  catalytic mechanism[13]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/" target="_blank" style="font-size:25px;">2011null</a> Detailed discussion of the chemical mechanism[14]</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/26414446" target="_blank" style="font-size:25px;">2015</a> Crystal structure[18]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/31959957" target="_blank" style="font-size:25px;">2020</a> Additional experiments to summarize the structure and function of VS ribozyme[20]</h4>
            </div>
          </div>
        </li>
    </ul>
</html>


## Description

***

<font size=4>Varkud satellite (VS) ribozyme is the largest known nucleolytic ribozyme and found to be embedded in VS RNA. VS RNA is a long non-coding RNA exists as a satellite RNA and is found in mitochondria of Varkud-1C and few other strains of Neurospora. VS ribozyme contains features of both catalytic RNAs and group 1 introns. VS ribozyme has both cleavage and ligation activity and can perform both cleavage and ligation reactions efficiently in the absence of proteins. VS ribozyme undergo horizontal gene transfer with other Neurospora strains. VS ribozymes have nothing in common with other nucleolytic ribozymes.</font><br>

<font size=4>VS RNA has a unique primary, secondary, and tertiary structure. The secondary structure of the VS ribozyme consists of six helical domains. Stem loop I forms the substrate domain while stem-loop II-VI forms the catalytic domain. When these 2 domains are synthesized in vitro separately, they can perform the self-cleavage reaction by trans-acting. The substrate binds into a cleft which is made by two helices. The likely active site of the ribozyme is a very important nucleotide A756. The A730 loop and A756 nucleotide are critical to its function since they participate in the phosphoric transfer chemistry activity of the ribozyme.</font>

## Structure

***
> 2D representation

**VS ribozyme catalyses site-specific phosphodiester cleavage by using a guanine, G638, and an adenine, A756, as a general base and acid, respectively.**

<table><tr>
<td>
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <title>Document</title>
</head>
<style>
   body {
    width: 100%;
    height: 100vh;
}
   button {
   margin-right: 0px;
}
   .main-container {
    display: flex;
    align-items: left;
    justify-content: center;
    height: 100%;
}
   .zoom-wrapper1 {
    width: 460px;
    height: 400px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper1">
            <div class="zoom-area1">
                <img src="https://www.ribocentre.org/images/VS_ribozymePic/VS2D.svg" alt="drawing" style="width:150px;height:110px" />
            </div>
        </div>
    </div>
    <script src="https://timmywil.com/panzoom/demo/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper1 = document.querySelector(".zoom-wrapper1");
      var panzoom1 = Panzoom(document.querySelector(".zoom-area1"), {
      maxScale: 6
      });
      zoomWraper1.addEventListener("wheel", panzoom1.zoomWithWheel);
      panzoom1.zoom(300 / document.querySelector(".zoom-area1 img").height);
      panzoom2.pan(0, 0);
      </script>
</td>
<td>
<link rel="stylesheet" type="text/css" href="https://www.ribocentre.org/css/fornac.css" media="screen" />
<div id='rna_ss' tabindex="1">
</div>

<meta charset="utf-8">
    <script type='text/javascript' src='https://www.ribocentre.org/js/jquery.js'></script>
    <script type='text/javascript' src='https://www.ribocentre.org/js/d3.js'></script>
    <script type='text/javascript' src='https://www.ribocentre.org/js/fornac.js'></script>
  
    <script type='text/javascript'>
        var container = new FornaContainer("#rna_ss", {'applyForce': 1,'editable':'true', 'initialSize':[450,450]});

        var options = {'structure':'((((((((((.....((((...((((.......))))..))))..((((((.(((..((((((..(((((.....)))))..(((((((.......)))))))......))).)))(((..(((.((((((((((....))))))))))...)))))).....)))))))))..))).)))))))',
                       'sequence':'GCGCUGUGUCGCAAUCUGCGAAGGGCGUCGUCGGCCCAAGCGGUAGUAAGCAGGGAACUCACCUCCAAUGAAACACAUUGUCGUAGCAGUUGACUACUGUUAUGUGAUUGGUAGAGGCUAAGUGACGGUAUUGGCGUAAGCCAAUACCGCAGCACAGCACAAGCCCGCUUGCGAGAUUACAGCGC'};

        container.addRNA(options.structure, options);
    </script>
</td>
</tr></table><br>


> 3D visualisation
<table><tr>
<td><img src="https://www.ribocentre.org/images/VS_ribozymePic/vs3D-1.png" alt="drawing" style="height:345px" border=0></td>
<td>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>PDBe Molstar</title>
    <!-- Molstar CSS & JS -->
    <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
    <script src="https://www.ebi.ac.uk/pdbe/pdb-component-library/js/pdbe-molstar-plugin-1.2.1.js"></script>
        <script>
        function customize()
        {
            viewerInstance.canvas.setBgColor({r:255, g:255, b:255})
        }
        </script>
  <style>
      * {
          margin: 0;
          padding: 0;
          box-sizing: border-box;
      }
      .msp-plugin ::-webkit-scrollbar-thumb {
          background-color: #ffffff !important;
      }
      .viewerSection {
        padding-top: 40px;
      }
      .controlsSection {
        width: 300px;
        float: left;
        padding: 40px 0 0 40px;
        margin-right: 30px;
      }
      .controlBox {
        border: 1px solid lightgray;
        padding: 10px;
        margin-bottom: 20px;
        background-color: #fff;
      }
      #myViewer{
        width:450px;
        height: 450px;
        position:relative;  
        
      }
      
  </style>
  </head>
  <body onload="customize()">
    <div id="myViewer">
  </div>
  <script>
    var viewerInstance = new PDBeMolstarPlugin();
    var options = {
      moleculeId: '4r4v',
      expanded: false,
      expanded: false,
      hideControls: true
      }
    var viewerContainer = document.getElementById('myViewer');
    viewerInstance.render(viewerContainer, options);
  </script>
  </body>
</html></td>
</tr></table><br>

> Catalytic centre

<table><tr>
<td><img src="https://www.ribocentre.org/images/VS_ribozymePic/vs-m1.png" alt="drawing" style="weight:600px;height:370px" border=0></td>
<td><img src="https://www.ribocentre.org/images/VS_ribozymePic/vs-m2.png" alt="drawing" style="weight:600px;height:370px" border=0></td>
</tr></table><br>


> Chemical mechanism

The VS ribozyme employs general acid-base catalysis, using the nucleobases of G638 (in its deprotonated form)[10] as general base to remove the proton from the 2’-OH nucleophile and A756 (in its protonated form)[4,7] as the general acid to protonate the leaving group. The pH - rate profile for the VS ribozyme is bell shaped, fitting pKa values of 5.2 and 8.4 corresponding to A756 and G638 respectively [10]. The acid and base were assigned by 5'-phosphorothiolate substitution experiments [13].  The pH dependence of a VS G638DAP ribozyme cleavage reaction indicated that proton transfer contributes at least 102-103 fold to the catalytic power of the ribozyme[10]. When the crystal structure of the VS ribozyme was eventually solved it was found that G638 and A756 were adjacent to the O2' and O5' atoms respectively, consistent with their proposed roles in catalysis[18]. Note that despite different overall RNA folds, the catalytic mechanisms of the VS and hairpin ribozymes are closely similar, both using G (general base) + A (general acid) mechanisms. 


## Reference

***

**[1] A site-specific self-cleavage reaction performed by a novel RNA in neurospora mitochondria.**<br>
Saville, B. J. and R. A. Collins <br>
[Cell 61(4): 685-696.(1990)](https://www.ncbi.nlm.nih.gov/pubmed/2160856)<br><br>

**[2] A secondary-structure model for the self-cleaving region of Neurospora VS RNA.**<br>
Beattie, T. L., J. E. Olive and R. A. Collins <br>
[Proc Natl Acad Sci U S A 92(10): 4686-4690.(1995)](https://www.ncbi.nlm.nih.gov/pubmed/7753865)<br><br>

**[3] Structure, folding and activity of the VS ribozyme : Importance of the 2-3-6 helical junction**<br>
D.A. Lafontaine, D.G. Norman and D.M.J. Lilley   <br>
[EMBO J. 20 1415-1424 (2001)](https://www.ncbi.nlm.nih.gov/pubmed/11250907)<br><br>

**[4] The A730 loop is an important component of the active site of the VS ribozyme.**<br>
Lafontaine, D. A., T. J. Wilson, D. G. Norman and D. M. Lilley <br>
[J Mol Biol 312(4): 663-674.(2001)](https://www.ncbi.nlm.nih.gov/pubmed/11575922)<br><br>

**[5] A pH controlled conformational switch in the cleavage site of the VS ribozyme substrate RNA.**<br>
Flinders, J. and T. Dieckmann <br>
[J Mol Biol 308(4): 665-679.(2001)](https://www.ncbi.nlm.nih.gov/pubmed/11350168)<br><br>

**[6] The global structure of the VS ribozyme.**<br>
D.A. Lafontaine, D.G. Norman and D. M.J. Lilley<br>
[EMBO J. 21, 2461-2471(2002)](https://www.ncbi.nlm.nih.gov/pubmed/nan)<br><br>

**[7] Functional Group Requirements in the Probable Active Site of the VS Ribozyme.**<br>
Lafontaine, D. A., T. J. Wilson, Z.-Y. Zhao and D. M. J. Lilley <br>
[Journal of Molecular Biology 323(1): 23-34.(2002)](https://www.ncbi.nlm.nih.gov/pubmed/12368096)<br><br>

**[8] Efficient, pH-dependent RNA ligation by the VS ribozyme in trans**<br>
A.C. McLeod and D.M.J. Lilley<br>
[Biochemistry 43, 1118 – 1125(2004)](https://www.ncbi.nlm.nih.gov/pubmed/nan)<br><br>

**[9] Nuclear magnetic resonance structure of the Varkud satellite ribozyme stem-loop V RNA and magnesium-ion binding from chemical-shift mapping.**<br>
Campbell, D. O. and P. Legault <br>
[Biochemistry 44(11): 4157-4170.(2005)](https://www.ncbi.nlm.nih.gov/pubmed/15766243)<br><br>

**[10] A guanine nucleobase important for catalysis by the VS ribozyme.**<br>
Wilson, T. J., A. C. McLeod and D. M. Lilley<br>
[EMBO J 26(10): 2489-2500.(2007)](https://www.ncbi.nlm.nih.gov/pubmed/17464286)<br><br>

**[11] The complete VS ribozyme in solution studied by small-angle X-ray scattering.**<br>
Lipfert, J., J. Ouellet, D. G. Norman, S. Doniach and D. M. Lilley <br>
[Structure 16(9): 1357-1367.(2008)](https://www.ncbi.nlm.nih.gov/pubmed/18786398)<br><br>

**[12] Formation of an active site in trans by interaction of two complete Varkud Satellite ribozymes**<br>
J. Ouellet, M. Byrne and D. M. J. Lilley<br>
[RNA 15, 1822-1826(2009)](https://www.ncbi.nlm.nih.gov/pubmed/nan)<br><br>

**[13] Nucleobase-mediated general acid-base catalysis in the Varkud satellite ribozyme.**<br>
Wilson, T. J., N. S. Li, J. Lu, J. K. Frederiksen, J. A. Piccirilli and D. M. Lilley<br>
[Proc Natl Acad Sci U S A 107(26): 11751-11756.(2010)](https://www.ncbi.nlm.nih.gov/pubmed/20547881)<br><br>

**[14] Do the hairpin and VS ribozymes share a common catalytic mechanism based on general acid-base catalysis ? A critical assessment of available experimental data.**<br>
T. J. Wilson and D. M. J. Lilley.  <br>
[RNA 17, 213-221 (2011)](https://www.ncbi.nlm.nih.gov/pubmed/nan)<br><br>

**[15] NMR structure of the A730 loop of the Neurospora VS ribozyme: insights into the formation of the active site.**<br>
Desjardins, G., E. Bonneau, N. Girard, J. Boisbouvier and P. Legault <br>
[Nucleic Acids Res 39(10): 4427-4437.(2011)](https://www.ncbi.nlm.nih.gov/pubmed/21266483)<br><br>

**[16] Nuclear magnetic resonance structure of the III-IV-V three-way junction from the Varkud satellite ribozyme and identification of magnesium-binding sites using paramagnetic relaxation enhancement.**<br>
Bonneau, E. and P. Legault <br>
[Biochemistry 53(39): 6264-6275.(2014)](https://www.ncbi.nlm.nih.gov/pubmed/25238589)<br><br>

**[17] The NMR structure of the II-III-VI three-way junction from the Neurospora VS ribozyme reveals a critical tertiary interaction and provides new insights into the global ribozyme structure.**<br>
Bonneau, E., N. Girard, S. Lemieux and P. Legault <br>
[RNA 21(9): 1621-1632.(2015)](https://www.ncbi.nlm.nih.gov/pubmed/26124200)<br><br>

**[18] Crystal structure of the Varkud satellite ribozyme.**<br>
Suslov, N. B., S. DasGupta, H. Huang, J. R. Fuller, D. M. Lilley, P. A. Rice and J. A. Piccirilli <br>
[Nat Chem Biol 11(11): 840-846.(2015)](https://www.ncbi.nlm.nih.gov/pubmed/26414446)<br><br>

**[19] Structural Basis for Substrate Helix Remodeling and Cleavage Loop Activation in the Varkud Satellite Ribozyme.**<br>
DasGupta, S., N. B. Suslov and J. A. Piccirilli <br>
[J Am Chem Soc 139(28): 9591-9597.(2017)](https://www.ncbi.nlm.nih.gov/pubmed/28625058)<br><br>

**[1] Confluence of theory and experiment reveals the catalytic mechanism of the Varkud satellite ribozyme.**<br>
Ganguly, A., B. P. Weissman, T. J. Giese, N. S. Li, S. Hoshika, S. Rao, S. A. Benner, J. A. Piccirilli and D. M. York <br>
[Nat Chem 12(2): 193-201.(2020)](https://www.ncbi.nlm.nih.gov/pubmed/31959957)<br><br>

